scripts/asm-installer/install_asm

```diff
@@ -2037,6 +2043,43 @@ register_gce_identity_provider() {
   retry 3 kubectl apply -f asm/identity-provider/googleidp.yaml
 }
 
+should_enable_service_mesh_feature() {
+  if [[ "${USE_VM}" -eq 0 ]]; then
+    false
+  fi
+}
+
+enable_service_mesh_feature() {
+  info "Enabling the service mesh feature..."
+
+  retry 2 run curl -s -H "Content-Type: application/json" \
+    -XPOST "https://gkehub.googleapis.com/v1alpha1/projects/${PROJECT_ID}/locations/global/features?feature_id=servicemesh"\
+    -d '{servicemesh_feature_spec: {}}' \
+    -K <(auth_header "$(get_auth_token)")
+}
+
+exit_if_service_mesh_feature_not_enabled() {
+  if ! is_service_mesh_feature_enabled; then
+    { read -r -d '' MSG; fatal "${MSG}"; } <<EOF || true
+The service mesh feature is not enabled on project ${PROJECT_ID}.
+Please run the script with the '--enable_gcp_components' flag to allow the
+script to enable it on your behalf.
+$(enable_common_message)
+EOF
+  fi
+}
+
+is_service_mesh_feature_enabled() {
+  local RESPONSE
+  RESPONSE="$(run curl -s -H "X-Goog-User-Project: ${PROJECT_ID}"  \
```

Comment:
Please add a comment with the IAM requirement "gkehub.features.get"

Reply:
done.